packages/cozy-konnector-libs/src/libs/saveBills.spec.js

```diff
@@ -0,0 +1,78 @@
+jest.mock('./saveFiles')
+jest.mock('./hydrateAndFilter')
+jest.mock('./addData')
+const addData = require('./addData')
+const hydrateAndFilter = require('./hydrateAndFilter')
+const saveFiles = require('./saveFiles')
+const saveBills = require('./saveBills')
+const manifest = require('./manifest')
+const logger = require('cozy-logger')
+const asyncResolve = val => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(val)
+    }, 1)
+  })
+}
+
+logger.setLevel('critical')
+
+beforeEach(async function() {
+  saveFiles.mockImplementation(entries => {
+    return asyncResolve(entries.map(doc => ({ ...doc, fileDocument: true })))
+  })
+  hydrateAndFilter.mockImplementation(entries => {
+    return asyncResolve(entries)
+  })
+  addData.mockImplementation(entries => {
+    return asyncResolve(entries)
+  })
```

Comment:
I don't know if I really understand your needs here, but I'm wondering if you can't use [`mockResolvedValue`](https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvaluevalue)?

Reply:
I don't this I can use it here since I want to use the input value as response

Reply:
Oh yes OK I see :+1: